<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticeWindow</class>
 <widget class="QMainWindow" name="PracticeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 245, 245);</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="verticalStackedWidgetPage1">
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>721</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(194, 214, 214);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>301</width>
            <height>281</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource.qrc">:/resources/images/study.svg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="homeNav">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>610</y>
            <width>191</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 10px;
border: 2px solid white;
background-color: transparent;
color: white;
}


QPushButton:hover {
	background-color: white;
color:  rgb(42, 74, 74);
}

QPushButton:pressed {
	background-color: #f0f0f0;
}</string>
          </property>
          <property name="text">
           <string>&lt; Back to Home</string>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
         <widget class="QStackedWidget" name="ChooseStack">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>290</y>
            <width>321</width>
            <height>291</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="groupBtn">
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>281</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>Choose a level below
 and start practising</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="easy">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>80</y>
              <width>191</width>
              <height>50</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius: 10px;
color: white;
background-color: #649696;
}


QPushButton:hover {
	
	background-color: rgb(85, 127, 127);
}

QPushButton:pressed {
	
	background-color: rgb(60, 90, 90);
}</string>
            </property>
            <property name="text">
             <string>Easy</string>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="medium">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>160</y>
              <width>191</width>
              <height>50</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius: 10px;
color: white;
	
	background-color: rgb(244, 175, 15);
}


QPushButton:hover {
	background-color: rgb(216, 152, 14);
}

QPushButton:pressed {
	
	
	background-color: rgb(177, 124, 11);
}</string>
            </property>
            <property name="text">
             <string>Medium</string>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="hard">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>240</y>
              <width>191</width>
              <height>50</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius: 10px;
color: white;
	background-color: rgb(172, 3, 3);
}


QPushButton:hover {
	
	background-color: rgb(148, 2, 2);
}

QPushButton:pressed {
	
	background-color: rgb(125, 2, 2);
}</string>
            </property>
            <property name="text">
             <string>Hard</string>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="result">
           <widget class="QLabel" name="score">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>0</y>
              <width>171</width>
              <height>71</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>48</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>100</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="level">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>80</y>
              <width>111</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(172, 3, 3);
border-radius: 5px;
color: white;</string>
            </property>
            <property name="text">
             <string>Hard</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QWidget" name="formLayoutWidget">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>130</y>
              <width>171</width>
              <height>91</height>
             </rect>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="rowWrapPolicy">
              <enum>QFormLayout::WrapLongRows</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="horizontalSpacing">
              <number>10</number>
             </property>
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: gray;</string>
               </property>
               <property name="text">
                <string>Duration:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="duration">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: gray;</string>
               </property>
               <property name="text">
                <string>00:00:00</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="time">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: gray;</string>
               </property>
               <property name="text">
                <string>23:00 12/10/2001</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="continueNav">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>240</y>
              <width>191</width>
              <height>50</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius: 10px;
background-color: #3E6E6E;
color: white;
}


QPushButton:hover {
	background-color: rgb(53, 94, 94);
}

QPushButton:pressed {
	background-color: rgb(31, 56, 56);
}</string>
            </property>
            <property name="text">
             <string>Continue</string>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>541</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Practice Result History</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QTableWidget" name="historyTable">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>555</width>
            <height>600</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableWidget::item { 
background-color: rgb(255, 255, 255);
border-bottom: 1px solid lightgray;
border-top: 10px solid white;
}





</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="autoScrollMargin">
           <number>10</number>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>135</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>135</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>40</number>
          </attribute>
          <column>
           <property name="text">
            <string>Time</string>
           </property>
           <property name="background">
            <color>
             <red>62</red>
             <green>110</green>
             <blue>110</blue>
            </color>
           </property>
           <property name="foreground">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Level</string>
           </property>
           <property name="background">
            <color>
             <red>62</red>
             <green>110</green>
             <blue>110</blue>
            </color>
           </property>
           <property name="foreground">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Duration</string>
           </property>
           <property name="background">
            <color>
             <red>62</red>
             <green>110</green>
             <blue>110</blue>
            </color>
           </property>
           <property name="foreground">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Score</string>
           </property>
           <property name="background">
            <color>
             <red>62</red>
             <green>110</green>
             <blue>110</blue>
            </color>
           </property>
           <property name="foreground">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
